apollo/src/graphql/storage-provider/storageProviderQueries.ts

```diff
@@ -13,13 +13,16 @@ builder.queryFields((t) => ({
         authScopes: {
             loggedIn: true,
         },
-        async resolve(root, args, ctx) {
+        async resolve(_root, _args, ctx) {
             const userTeams = await ObjectionTeamEdge.query()
                 .where({ userId: ctx.user.$id() })
                 .select(['teamId']);
 
             const storageProvider = await ObjectionStorageProvider.query()
-                .whereIn('projects.teamId', userTeams.map(edge => edge.teamId))
```

Comment:
Aw shucks pretty sure this was just a careless copy pasta on my part. Thanks for catching it!

Reply:
No problemo!